iOS/WAStickersThirdParty/StickerPack.swift

```diff
@@ -16,6 +16,7 @@ enum StickerPackError: Error {
     case emptyString
     case unsupportedImageFormat(String)
     case imageTooBig(Int64, Bool)
+    case imageTooSmall
```

Comment:
I think that something like "invalidImage" would be better since the image isn't small: there is no image :)